routing/routing_tests/road_graph_builder.cpp

```diff
@@ -88,7 +88,7 @@ double RoadGraphMockSource::GetMaxSpeedKMPH() const
 void RoadGraphMockSource::ForEachFeatureClosestToCross(m2::PointD const & /* cross */,
                                                        ICrossEdgesLoader & edgesLoader) const
 {
-  for (size_t roadId = 0; roadId < m_roads.size(); ++roadId)
+  for (uint32_t roadId = 0; roadId < m_roads.size(); ++roadId)
```

Comment:
Why? m_roads.size() is size_t, why not  to cast size_t to uint32_t when you call MakeTestFeatureId?